README.md

```diff
@@ -21,7 +21,7 @@ Endless Sky has very minimal system requirements, meaning most systems should be
 |Storage Free | 250 MB | 1 GB |
 
 ## Building from source
-Most development is done on Linux and Windows, using the [SCons](https://scons.org/) build tool to compile the project. For those wishing to use an IDE, project files are provided for [Code::Blocks](https://www.codeblocks.org/) to simplify the project setup. There is also a CMake project available ([build instructions](readme-cmake.md)).
+Most development is done on Linux and Windows, using CMake ([build instructions](readme-cmake.md)) to compile the project. For those wishing to use an IDE, project files are provided for [Visual Studio](https://visualstudio.microsoft.com/) and [Code::Blocks](https://www.codeblocks.org/) to simplify the project setup. [SCons](https://scons.org/) was the primary build tool up until 0.9.16, and some files and information continue to be available for it.
```

Comment:
```suggestion
Most development is done on Linux and Windows, using CMake ([build instructions](readme-cmake.md)) to compile the project. For those wishing to use an IDE, project files are provided for [Code::Blocks](https://www.codeblocks.org/) to simplify the project setup, and other IDEs are supported through their respective CMake integration. [SCons](https://scons.org/) was the primary build tool up until 0.9.16, and some files and information continue to be available for it.
```

What do you think? There are no project files for Visual Studio, it's all done through CMake (you can imagine that the CMake files are the project files, which is effectively what happens). Maybe my wording is not that great though.

Maybe you should link to the CMake website for the first link, and then change the link in the second paragraph to the CMake instructions?